product_supplierinfo_group/models/product_supplierinfo_group.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2012 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+# Copyright (C) 2015-Today GRAP (http://www.grap.coop)
+# Copyright (C) 2020-Today Akretion (http://www.akretion.com)
+# @author Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License GPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductSupplierinfoGroup(models.Model):
+    _name = "product.supplierinfo.group"
+    _description = "Supplierinfo group"
+
+    product_tmpl_id = fields.Many2one("product.template", required=True)
+    supplierinfo_ids = fields.One2many("product.supplierinfo", "supplierinfo_group_id")
+    product_id = fields.Many2one(
+        "product.product",
+        "Product Variant",
+        help="If not set, the vendor price will apply to all "
+        "variants of this product.",
+    )
+    name = fields.Many2one(
```

Comment:
I know that odoo use "name" for the link to the partner but as it's an new object I prefers to use "partner_id" here.

Reply:
Agree

Reply:
done